src/routes/api/v1/search-index/+server.ts

```diff
@@ -100,3 +107,29 @@ export const POST = async ({ request, locals: { contentfulClient } }) => {
     throw error(400, message as string);
   }
 };
+
+// Return all suitable Page Metadata, News, and Events map items
+export const GET = async ({ locals: { contentfulClient } }) => {
```

Comment:
`TODO` for my service entry updates: add a query param to quickly filter between `metadata`, `news-and-events`, and (from mine) `service-entries`